packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts

```diff
@@ -156,16 +158,15 @@ export const getVideoReceiverMqa = ({videoReceiver, statsResults, lastMqaDataSen
     statsResults[mediaType][sendrecvType].totalPacketsReceived - lastPacketsReceived || 0;
   videoReceiver.streams[0].common.rtpPackets = videoReceiver.common.rtpPackets;
 
-  const totalPacketLoss =
-    statsResults[mediaType][sendrecvType].totalPacketsLost - lastPacketsLost || 0;
-
-  // Hope by hop are numbers and not percentage so we compare on what we sent the last min
+  // Hop by hop are numbers and not percentage so we compare on what we sent the last min
   // this is including packet lost
-  videoReceiver.common.mediaHopByHopLost = totalPacketLoss;
-  videoReceiver.common.rtpHopByHopLost = totalPacketLoss;
+  const totalPacketsLost =
```

Comment:
Could we just make all this totalPacketsLost stuff a function and hand it a receiver as well as setting the common.mediaHopByHopLost ? it seems like there's some redundancy between this and the audio stuff... obviously not required, but it seems like an easy thing to do and then have a separate util function and a smaller test for it

Reply:
There's probably a lot more than just this where there's redundancy between `audioReceiver` and `videoReceiver` (and even more between `audioSender` and `videoSender`). I agree that it would be great to have this logic in a common location/function shared across media types... though if we do this, we should probably do it all in one go (i.e. instead of having separate `getAudioReceiverMqa` and `getVideoReceiverMqa` functions, have a single `getReceiverMqa` function that handles both). Honestly, that's probably a pretty big change that's out-of-scope for this PR... I'd suggest doing it as a separate task.